modules/core/src/main/java/org/apache/ignite/internal/processors/platform/PlatformProcessor.java

```diff
@@ -47,6 +48,13 @@
      */
     public PlatformContext context();
 
+    /**
+     * Returns a value indicating whether {@link #context()} is available.
+     *
+     * @return value indicating whether {@link #context()} is available.
+     */
+    public boolean hasContext();
```

Comment:
I don't think this is a good name for this method as it does not say anything for person reading the code. I actually had to go to implementation of `context()` method in `PlatformNoopProcessor` to understand it's meaning. Maybe `hasPlatformRuntime()` or `isPlatformNode()` would fit better here?